tf_helpers/lax_tests.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
To follow the NT convention, could you rename this file to `lax_test.py`, and put it into the `tests/` folder?